action.yaml

```
@@ -115,6 +119,7 @@ runs:
 
     - name: Create commit
       id: commit
+      if: inputs.github-app-id && inputs.github-app-private-key
```

The Create PR step should probably have the same `if`

---

Yeah. Should we have it fail loudly if `publish` is `true` and credentials aren't provided though?

---

Opted to add a pre-emptive check for such a case